proto/buf/alpha/registry/v1alpha1/doc.proto

```diff
@@ -87,12 +108,11 @@ message GetModuleDocumentationResponse {
 // ModuleDocumentation provides the name of the module and associated documentation.
 message ModuleDocumentation {
   string name = 1;
-  repeated string packages = 2;
   // This is the string representation of the contents of the buf.md file for module-level documentation.
   //
   // The buf.md file is a part of the module.
   // string is used to enforce UTF-8 encoding or 7-bit ASCII text.
-  string documentation = 3;
+  string documentation = 2;
```

Comment:
Changing tags is a breaking change - you can delete a field, but never change a fields' tag

Reply:
Oh yeah, I know we used to be down for breaking changes, but now that our API is public, should we not do that?

Reply:
It will just make the frontend update harder, but yea if you're willing to take that on, that's fine

Reply:
I think we can just keep the tag and avoid the breaking change